packages/react-components/react-list-preview/stories/List/SingleActionSelectionDifferentPrimary.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import { Persona, SelectionItemId } from '@fluentui/react-components';
+import { List, ListItem } from '@fluentui/react-list-preview';
+
+import * as React from 'react';
+const names = [
+  'Melda Bevel',
+  'Demetra Manwaring',
+  'Eusebia Stufflebeam',
+  'Israel Rabin',
+  'Bart Merrill',
+  'Sonya Farner',
+];
+
+type Item = {
+  name: string;
+  id: string;
+  avatar: string;
+};
+
+const items: Item[] = names.map(name => ({
+  name,
+  id: name,
+  avatar:
+    'https://res-1.cdn.office.net/files/fabric-cdn-prod_20230815.002/office-ui-fabric-react-assets/persona-male.png',
+}));
```

Comment:
This can be simplified to:

```suggestion
const items = [
  'Melda Bevel',
  'Demetra Manwaring',
  'Eusebia Stufflebeam',
  'Israel Rabin',
  'Bart Merrill',
  'Sonya Farner',
].map(name => ({
  name,
  id: name,
  avatar:
    'https://res-1.cdn.office.net/files/fabric-cdn-prod_20230815.002/office-ui-fabric-react-assets/persona-male.png',
}));
```